### ISSUE TITLE
fltkhs-make-tree shows just one item

### ISSUE THREAD

**Issue by simonmichael** [deech/fltkhs-demos#2]
I saw only a root item in the list probably it's supposed to have some children ?

**Comment by deech** [comment]
Yes that demo is broken. In fact I'm going to remove it, `flkths-tree-simple` serves the same purpose. Thanks!
